### PR TITLE
Transactions advanced filtering

### PR DESCRIPTION
**Pull request by BMartinos**
This PR relates to issue:  #256 and #243 

I have also made a few small additions:
Sidebar menu padding smaller
new loading gif
default limit option for user defined settings

### CODE REVIEW

**Comment by BMartinos** [comment]
Hey @devcritter Can you review this PR for me when you get a chance. Thanks

**Comment by devcritter** [comment]
You should add some validation on the "status" box - currently you can type in anything and break the request. The length can be restricted to 3, and maybe add a quick regex? Something like `/\d(\d\d|xx)/`

File: app/scripts/services/rest.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ angular.module('openhimConsoleApp')
       }),
 
       Transactions: $resource( server + '/transactions/:transactionId', { transactionId: '@_id' }),
+      TransactionsFilterOptions: $resource( server + '/transactions-filter-options/', {}),
```
Comment (devcritter) [inline app/scripts/services/rest.js]: This needs to be removed

**Comment by devcritter** [comment]
Thanks @BMartinos ! This looks good and works really well. Just some comments, but after that :+1:

**Pull request merged**